TaskManager/Extensions/String.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  String.swift
```

Comment:
Let's rename the file to String+Extensions

Reply:
file duplicated, lets delete it

Reply:
Look for duplicate files in project folder in finder, they may not appear in xcode project